### ISSUE TITLE
About the ECO_HC tracker speed problem.

### ISSUE THREAD

**Issue by fybaft2012** [martin-danelljan/ECO#24]
Hi MD,
Thanks for you great work! One thing I need your help, I tested your code on my PC, which is of 24 GB memory, Intel i7-6700 CPU with 3.40GHz x 8, and NVIDIA TITAN X.
When I run ECO_HC file, the speed is only about 20 fps - 30fps, but in the paper said it can run at 60 fps, There are big difference between the two, I want to know why it happened and how should I do to run it at 60 fps.
Sorry for disturbing you and hope you can help me with it.
Thanks!

**Comment by goutamgmb** [comment]
Hi,

Did you run with params.use_gpu set to false? The ECO_hc code is not optimized for gpu and hence can be slower on gpu due to additional memory transfers.

Secondly, set params.visualization and params.debug to false.

**Comment by fybaft2012** [comment]
Hi,
Thank you for answering. Yes bro, I set params.use_gpu = false, params.visualization and params.debug to false already, and the speed is still 23.77 fps then. With OBT sequence 'Bolt'.

**Comment by goutamgmb** [comment]
That is strange. We got over 65 fps on the Bolt sequence. ( Intel Xeon E5-1620 CPU was used for the benchmarking in the paper)

Did you get this warning message? 
'Error when using the mexResize function. Using Matlab''s interpolation function instead, which is slower'

**Comment by fybaft2012** [comment]
Yes, you are right,  I got that warning message, I think because I think the compiled mexResize which you provided does not work on my computer, is it because of that?  If so, would you please give me the advice how I can compile my own mexResize file on my pc? By the way, I use Ununtu 16.04 and opecv 2.4 v5, and gcc 4.9.4.
Thanks!

**Comment by goutamgmb** [comment]
Yes, not using mexResize can have significant impact on speed. Have you tried compiling mexResize using external_libs/mexResize/compile.m?

**Comment by fybaft2012** [comment]
Yes, bro. I have tried to compile the mexResize function and it said that it is compiled successfully, but when I use the re-compiled one, it still has that warning message information. I am not sure it is because of the opencv path settings or other reasons?
Thank you very much for your patient answer!

**Comment by fybaft2012** [comment]
Hi goutamgmb,
About the mexResize compiling problem, I am still confused. When I use the re-complied mexResize file compiled using openCV 2.4.9 on my Ubuntu, it show it is a Invalid MEX-file mexResize.mexa64. is that because of openCV version issue ? If so, which opencv version I need to use exactly on my ubuntu?
Looking forward your help!
